src/igvfd/audit/measurement_set.py

```diff
@@ -224,82 +224,8 @@ def audit_missing_institutional_certification(value, system):
-
-
-@audit_checker('MeasurementSet', frame='embedded')
-def audit_Variant_FlowFISH_missing_variant_sequencing_auxiliary_set(value, system):
-    '''
-    [
-        {
-            "audit_description": "Variant FlowFISH measurement sets are expected to link to a variant sequencing auxiliary set.",
-            "audit_category": "missing auxiliary set",
-            "audit_level": "NOT_COMPLIANT"
-        }
-    ]
-    '''
-    description = get_audit_description(audit_Variant_FlowFISH_missing_variant_sequencing_auxiliary_set)
-    if value.get('preferred_assay_title') == 'Variant FlowFISH':
-        auxiliary_sets = [system.get('request').embed(auxiliary_set['@id'], '@@object?skip_calculated=true')
-                          for auxiliary_set in value.get('auxiliary_sets', '')]
-        if not (auxiliary_sets) or not ([auxiliary_set for auxiliary_set in auxiliary_sets if auxiliary_set.get('file_set_type') == 'variant sequencing']):
-            detail = (
-                f'Measurement set {audit_link(path_to_text(value["@id"]),value["@id"])} '
-                f'has no variant sequencing `auxiliary_sets`.'
-            )
-            yield AuditFailure('missing auxiliary set', f'{detail} {description}', level='NOT_COMPLIANT')
-
-
-@audit_checker('MeasurementSet', frame='embedded')
-def audit_CRISPR_assay_missing_cell_sorting_auxiliary_set(value, system):
-    '''
-    [
-        {
-            "audit_description": "CRISPR-based measurement sets that utilize flow cytometry are expected to link to a cell sorting auxiliary set.",
-            "audit_category": "missing auxiliary set",
-            "audit_level": "NOT_COMPLIANT"
-        }
-    ]
-    '''
-    description = get_audit_description(audit_CRISPR_assay_missing_cell_sorting_auxiliary_set)
-    assay_term_name = value.get('assay_term', {}).get('term_name')
-    preferred_assay_title = value.get('preferred_assay_title', '')
-    crispr_flow_assays = ['CRISPR perturbation screen followed by flow cytometry and FISH', 'Variant FlowFISH']
-    if assay_term_name in crispr_flow_assays or preferred_assay_title in crispr_flow_assays:
-        auxiliary_sets = [system.get('request').embed(auxiliary_set['@id'], '@@object?skip_calculated=true')
-                          for auxiliary_set in value.get('auxiliary_sets', '')]
-        if not (auxiliary_sets) or not ([auxiliary_set for auxiliary_set in auxiliary_sets if auxiliary_set.get('file_set_type') == 'cell sorting']):
-            detail = (
-                f'Measurement set {audit_link(path_to_text(value["@id"]),value["@id"])} '
-                f'has no cell sorting `auxiliary_sets`.'
-            )
-            yield AuditFailure('missing auxiliary set', f'{detail} {description}', level='NOT_COMPLIANT')
-
-
 @audit_checker('MeasurementSet', frame='object')
-def audit_missing_auxiliary_sets(value, system):
+def audit_missing_auxiliary_set_link(value, system):
```

Comment:
This audit concerns a different issue than the rest of the "missing auxiliary set" audits, so I did not include it in the unifying audit below.